packages/react/src/hooks/asset/useCreateAsset.ts

```diff
@@ -26,12 +26,16 @@ export function useCreateAsset<TLivepeerProvider extends LivepeerProvider>(
 ) {
   const livepeerProvider = useLivepeerProvider<TLivepeerProvider>();
 
-  const [uploadProgress, setUploadProgress] = useState(0);
+  const [uploadProgress, setUploadProgress] = useState<number | undefined>(
+    undefined,
+  );
+
   const internalQuery = useInternalMutation(
     async (args: CreateAssetArgs) =>
       createAsset<TLivepeerProvider>({
         ...args,
-        onUploadProgress: setUploadProgress,
+        onUploadProgress: (progress) =>
+          setUploadProgress(progress === 1 ? undefined : progress),
```

Comment:
How do we know when the upload is complete again? Should we clear the progress when that happens?

Reply:
Yeah when I was playing around with it, it made a little more sense for the `uploadProgress` field to be undefined when there's no upload in progress. It made that progressFormatted conditional a bit easier to handle. So once it hits 100% it'll be undefined

Reply:
OK sounds good :)